config/application.php

```diff
@@ -39,7 +39,6 @@
  */
 define('WP_HOME', str_replace('https://', 'http://', env('WP_HOME')));
 define('WP_SITEURL', str_replace('https://', 'http://', env('WP_SITEURL')));
-define('FORCE_SSL_ADMIN', true);
```

Comment:
Is this handled somewhere in the NGINX config then?

Reply:
This is simply a WP admin force. SSL is handled by the server. It has an adverse effect on local dev which is why it is removed - dev is always http. Is that OK?